src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/ApplyChangesToWorkspaceContext.cs

```diff
@@ -137,7 +137,7 @@ protected override void Initialize()
 
             foreach (ExportLifetimeContext<IWorkspaceContextHandler> handler in _handlers)
             {
-                handler.Value.Initialize(_context);
+                handler.Value.Initialize(_context!);
```

Comment:
This might not be right?  Maybe need to annotate `Initialize` as taking possible nulls?  @davkean ?